web/models/setting/system_variable.ex

```diff
@@ -79,6 +73,26 @@ defmodule CoursePlanner.SystemVariable do
   end
   defp validate_value_type(changeset), do: changeset
 
+  def parse_value(value, type) do
+    case type do
+      "string"  -> {:ok, value}
+      "list"     -> parse_list(value)
+      "integer" -> parse_integer(value)
+      "boolean" -> parse_boolean(value)
+      _         -> {:error, "unknown type"}
+    end
+  end
+
+  def parse_list(value) do
+    parse_list =
+    value
```

Comment:
if it is a simple one it is fine, if it needs to get more complicate then leave it as it is :)